mne/cov.py

```diff
@@ -962,8 +1109,59 @@ def _compute_covariance_auto(data, method, info, method_params, cv,
     logliks = np.array([_cross_val(data, e, cv, n_jobs) for e in estimators])
 
     # undo scaling
-    for c in estimator_cov_info:
-        _undo_scaling_cov(c[1], picks_list, scalings)
+    for cov in estimator_cov_info:
+        # this is now a bit convoluted.
+        # The right part of the cov has to be picked, reprojected if needed
+        # and here the clipping has to happen.
+        # XXX we need a rank dict here and clip conditional upon
+        # low rank config.
+        if has_sss:
+            # XXX the logic is broken here for EEG
```

Comment:
This looks a bit like something we need to raise an error or warning for, or fix before release

Reply:
I should remove the XXX now....